reports/Oregon.md

```diff
@@ -3475,10 +3475,69 @@ id: or-portland-213
 * https://twitter.com/hungrybowtie/status/1288734291334053888
 
 
+### Police charge, tackle, and pepper spray protesters | August 1st
+
+Police gave directions to protesters to walk in a certain direction. Protesters followed directions. Police charged at protestors, shoving many to the ground. One protester on a bicycle is knocked over. The protester lands on top of the bike and has trouble getting up. Police pepper spray protesters as well.
+
+tags: protester, tackle, shove, grab, strike, pepper-spray, spray
+
+id: or-portland-225
+
+**Links**
+
+* https://twitter.com/hungrybowtie/status/1289791877223350273
```

Comment:
It's really tough to see, but I think a cop maces a protester on the ground here. Might want to mention this if that's what's happening.